np-devops-team1/DevOps_Main
# ref: convert main codebase to python package

## Description

Add __init__.py file to allow python to export the main codebase as a package and allow functions to be imported

![image](https://user-images.githubusercontent.com/33172738/148003261-1208f710-6915-4346-bc71-11a3404078a9.png)

Given the file structure shown above, functions can be directly imported and tested from the submodule as a python package using 
```py
from DevOps_Main import fileName

assert fileName.functionHere() == true
```

## Pytest Configuration

To better facilitate pytest with vscode, follow the instructions below to ensure test explorer can properly identify tests
1. Open vscode commands ([CTRL + SHIFT + P] by default)
2. Search for `PythonL Configure Tests`
3. Select pytest as the framework
4. Select root directory as location containing tests
